content/doc/features/authentication.en-us.md

```diff
@@ -95,6 +95,16 @@ Both the LDAP via BindDN and the simple auth LDAP share the following fields:
     matching supplied login name against multiple attributes such as user
     identifier, email or even phone number.
   - Example: `(&(objectClass=Person)(|(uid=%[1]s)(mail=%[1]s)(mobile=%[1]s)))`
+- Enable user synchronization
+  - This option enables a periodic task that keeps the Gitea users synchronized
+    with the LDAP server. The default period is every 24 hours but that can be
+    changed in the
+    [app.ini](https://github.com/go-gitea/gitea/blob/master/conf/app.ini) file
```

Comment:
The new path to this file is: https://github.com/go-gitea/gitea/blob/master/custom/conf/app.ini.sample